core/template.js

```diff
@@ -13,10 +13,18 @@
 		rePlaceholder = /{([^}]+)}/g,
 		reEscapableChars = /([\\'])/g,
 		reNewLine = /\n/g,
-		reCarriageReturn = /\r/g;
+		reCarriageReturn = /\r/g,
+		isIndirectEvalSupported = true;
+
+	try {
+		Function( '1' )();
```

Comment:
Just want to mention that testing like this could end up cluttering the console with CSP violation reports.